docs/sources/configure/_index.md

```diff
@@ -2824,6 +2824,11 @@ The `limits_config` block configures global and per-tenant limits in Loki.
 # CLI flag: -validation.increment-duplicate-timestamps
 [increment_duplicate_timestamp: <boolean> | default = false]
 
+# List of labels to map to service_name label, if not present in the stream
+# labels during ingestion. Empty list disables setting the label.
+# CLI flag: -validation.discover-service-name
+[discover_service_name: <list of strings> | default = [service app application name app_kubernetes_io_name container container_name component workload job]]
```

Comment:
Suggestion that would make it clearer that the label will be one of the items in the list, not the entire list.

```suggestion
[discover_service_name: <list of strings> | default = [One from service, app, application name, app_kubernetes_io_name, container, container_name component, workload, OR job]]
```

Reply:
The part you have modified is auto-generated. I think it would be hard to update that part. Can we update the help text instead?

Reply:
Sounds like a good workaround to me!